### PR TITLE
Simple Bosses #6 Jingu Mastery and #7 Desolate

### PR DESCRIPTION
**Pull request by Minnakht**
From #870 and #871 

Boss 6
Unit name: `npc_dota_boss_simple_6`
Ability 1: `boss_jingu_mastery`
Ability 2: `boss_inner_beast`
Ability 3: `boss_resistance`

The second ability is kludged from Beastmaster's Inner Beast to allow finer control over the boss's attack speed.

Boss 7
Unit name: `npc_dota_boss_simple_7`
Ability 1: `boss_desolate`
Ability 2: `boss_resistance`

hi I'm pretending to be Chrono

I hope temporary in-game display names aren't minded

### CODE REVIEW

File: game/resource/English/ability/units/boss/tooltip_as_passive_6.txt [hunk 1]
```diff
@@ -0,0 +1,8 @@
+//=============================================================================
+// Simple Boss Desolate
+//=============================================================================
+"DOTA_Tooltip_ability_boss_as_passive_6"                      "Attack Speed Bonus"
+"DOTA_Tooltip_ability_boss_as_passive_6_Lore"                  "Roshan has been using this since 6.86."
+"DOTA_Tooltip_ability_boss_as_passive_6_Description"                "This unit's increased attack speed offsets its low base attack time. You can't slow its attacks as easily."
+"DOTA_Tooltip_ability_boss_as_passive_6_radius"            "RADIUS:"
```
Comment (Trildar) [inline game/resource/English/ability/units/boss/tooltip_as_passive_6.txt]: Why have the radius in the tooltip when we don't actually want it to be an aura?

Comment (Minnakht) [inline game/resource/English/ability/units/boss/tooltip_as_passive_6.txt]: I have approximately 0.023 of an idea what I'm doing and this is how Return handles potentially turning into an aura

File: game/resource/English/ability/units/boss/tooltip_boss_jingu_mastery.txt [hunk 2]
```diff
@@ -0,0 +1,12 @@
+//=============================================================================
+// Simple Boss Jingu Mastery
+//=============================================================================
+"DOTA_Tooltip_ability_boss_jingu_mastery"                      "Jingu Mastery"
+"DOTA_Tooltip_ability_bboss_jingu_mastery_Lore"                  "Always sensitive to the his own moods, the Jingu Banger radiates power when his combat fervor is on full display."
```
Comment (Trildar) [inline game/resource/English/ability/units/boss/tooltip_boss_jingu_mastery.txt]: Typo in key "bboss"

File: game/scripts/npc/abilities/boss/boss_jingu_mastery.txt [hunk 3]
```diff
@@ -0,0 +1,55 @@
+"DOTAAbilities"
+{
+  //=================================================================================================================
+  // Monkey King: Jingu Mastery
+  //=================================================================================================================
+  "monkey_king_jingu_mastery"
```
Comment (Trildar) [inline game/scripts/npc/abilities/boss/boss_jingu_mastery.txt]: You forgot to rename this.

Comment (Minnakht) [inline game/scripts/npc/abilities/boss/boss_jingu_mastery.txt]: travis told me, already fixed in branch. How do I resubmit

Comment (imaGecko) [inline game/scripts/npc/abilities/boss/boss_jingu_mastery.txt]: Change files, commit, push. That's all

Comment (Trildar) [inline game/scripts/npc/abilities/boss/boss_jingu_mastery.txt]: Reopen this PR. You just have to push commits to the branch this is on.

File: game/resource/English/ability/units/boss/tooltip_boss_jingu_mastery.txt (hunk 2 above)
Comment (Minnakht) [inline game/resource/English/ability/units/boss/tooltip_boss_jingu_mastery.txt]: nice catch, amended

File: game/resource/English/ability/units/boss/tooltip_as_passive_6.txt (hunk 1 above)
Comment (Trildar) [inline game/resource/English/ability/units/boss/tooltip_as_passive_6.txt]: You don't need to have tooltips for every value if it's not really helpful. Saying "RADIUS: 0" will just look weird.

File: game/scripts/npc/abilities/boss/boss_as_passive_6.txt [hunk 4]
```diff
@@ -0,0 +1,31 @@
+"DOTAAbilities"
+{
+  //=================================================================================================================
+  // Boss: Attack Speed Passive
+  //=================================================================================================================
+  "boss_as_passive_6"
```
Comment (Chronophylos) [inline game/scripts/npc/abilities/boss/boss_as_passive_6.txt]: just call this `boss_inner_beast`

File: game/resource/English/ability/units/boss/tooltip_as_passive_6.txt (hunk 1 above)
Comment (Minnakht) [inline game/resource/English/ability/units/boss/tooltip_as_passive_6.txt]: noted, will remove.

**Comment by Chronophylos** [comment]
Now you need to break something if you're pretending to be me

File: game/scripts/npc/npc_abilities_custom.txt [hunk 5]
```diff
@@ -5,6 +5,9 @@
 #base "abilities/boss/boss_juxtapose.txt"
 #base "abilities/boss/boss_resistance.txt"
 #base "abilities/boss/boss_fury_swipes.txt"
+#base "abilities/boss/boss_desolate.txt"
+#base "abilities/boss/boss_jingu_mastery.txt"
+#base "abilities/boss/boss_as_passive_6.txt"
```
Comment (chrisinajar) [inline game/scripts/npc/npc_abilities_custom.txt]: Where is this file?

Comment (Minnakht) [inline game/scripts/npc/npc_abilities_custom.txt]: nowhere right now, admittedly. hol up imma change that name

**Pull request merged**